man/oci-image-tool-create.1.md

```diff
@@ -1,8 +1,8 @@
-% OCI(1) OCI-CREATE-RUNTIME-BUNDLE User Manuals
+% OCI(1) OCI-IMAGE-TOOL User Manuals
```

Comment:
I think we want `OCI-IMAGE-TOOL-CREATE(1) OCI Image Tool User Manuals` here to match the pattern set in Git's `GIT-INIT(1)  Git Manual` (in `git-init(1)`)